whitelabelco/sofo-foods
# Cp/feat/remove some product pages

### Description 💻

Resolves: #173 

### Implementation 💡

Client no longer wants the following products to appear on the Products page or as individual product pages:
- Proprietary Items
- Miscellaneous
- Frozen/Fresh refrigerated

#### Type of change


### How to test 🔍

- `npm i`
- `npm run dev`
- navigate to `http://localhost:3000/`
- click "Products"
- confirm the above products no longer appear on the sidebar or image grid

### Screenshots/Demos 📷

### Checklist ✅
